### ISSUE TITLE
Accessibility: the &lt;button&gt; used with account sign-in needs role="none"

### ISSUE THREAD

**Issue by MikeTschudi** [Esri/global-nav#304]
When the account sign-in dropdown is open, the element
`<button class="esri-header-canvas" tabindex="-1" data-open="true" data-state="menu"></button>`
is flagged by the [Accessibility Insights for Web](https://accessibilityinsights.io/docs/en/web/overview) tool as needing "discernible text".

Since the button doesn't have an accessibility role, suggest adding `role="none"` to element.

**Comment by MikeTschudi** [comment]
What is the purpose of this button, please? If I remove it from the app, nothing appears to have changed except for the cleaning up of some a11y issues.

**Comment by paulcpederson** [comment]
canvas is the semi-opaque black "scrim" which should appear behind the open menu. I don't know why the author of this lib made it a button, but it's a design element that shouldn't be removed entirely.

**Comment by MikeTschudi** [comment]
Thank you, @paulcpederson; I see its value now.

The source creates it as a `div`
```
const $headerCanvas = $('div', {
  class: 'esri-header-canvas',
  tabindex: '-1',
  data: { open: false }
});
```
but the minimized library creates it as a `button`
```
o = K("button", {
    class: "esri-header-canvas",
    tabindex: "-1",
    data: { open: !1 }
})
```
Perhaps we can add text such as "scrim" to the button. This removes the "A button is empty or has no value text." error displayed by the [WAVE Web Accessibility Evaluation Tool](https://wave.webaim.org/) and does not appear visually or aurally (with NVDA). Or perhaps it's a false negative from WAVE--shouldn't `role="none"` remove it from WAVE's checking? The [Accessibility Insights for Web](https://accessibilityinsights.io/docs/en/web/overview) tool doesn't raise an "empty button" error, e.g.
